src/main/java/ch/njol/skript/effects/EffSendTitle.java

```diff
@@ -57,6 +57,10 @@
 					"send subtitle %string% [to %players%]");
 		
 	}
+
+	private final static int DEFAULT_FADEIN = 10;
+	private final static int DEFAULT_STAY = 70;
+	private final static int DEFAULT_FADEOUT = 20;
```

Comment:
Can you put these above the static block?

Reply:
Turned out they aren't needed, because passing -1 to `sendTitle` method sets the times to the defaults.